### PR TITLE
Adding a logging framework for more detailed logging of vRA Terraform plugging.

### PR DESCRIPTION
**Pull request by Prativa20**

### CODE REVIEW

File: utils/contants.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package utils
+
+const (
+	REQUEST_STATUS = "request_status"
+
```
Comment (raghavav) [inline utils/contants.go]: Too many empty lines here. Delete please.

File: utils/logger.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+func InitLog() {
+	initTerraformVraProviderLog()
+}
+
+func initTerraformVraProviderLog() {
+	//Setup terraform service log
+	var backendList = []logging.Backend{}
+	var logFile *os.File
+	logFile, err := os.OpenFile("./"+terraformVraProviderFileName, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
```
Comment (raghavav) [inline utils/logger.go]: I'd be more comfortable if someone could test this on a Windows system. For example, the forward slash here worries me.

**Review by raghavav — CHANGES_REQUESTED** [review]
Some comments. Mainly, need to resolve the failing tests in CI, and test it on Windows.

**Review by raghavav — APPROVED** [review]

**Comment by Prativa20** [comment]
Tested on windows as well, works fine!

**Pull request merged**